src/rules/__tests__/no-missing-translations.test.ts

```diff
@@ -0,0 +1,82 @@
+import { ESLintUtils } from "@typescript-eslint/utils"
+import noMissingTranslations from "../no-missing-translations"
+import { jest } from "@jest/globals"
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: "@typescript-eslint/parser",
+})
+jest.mock("fs", () => {
+  const actualFs = jest.requireActual<typeof import("fs")>("fs")
+  const newFs = {
+    ...actualFs,
+    readFileSync: jest.fn((file: string) => {
+      if (file === "en.json") {
+        return JSON.stringify({
+          "Existing key": "Existing value",
```

Comment:
Beetje gekke inhoud van een translation file. Dit kan ook:
```suggestion
          "Existing key": "Existing key",
```
en dan in het Nederlands:
```
          "Existing key": "Bestaande sleutel",
```